_security/configuration/opensearch-keystore.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: OpenSearch keystore
+parent: Configuration
+nav_order: 50
+---
+
+# OpenSearch Keystore
+
+`opensearch-keystore` is a utility script for managing the OpenSearch keystore. This keystore provides a secure method to store sensitive information, such as passwords and keys, used in the OpenSearch cluster. The script allows users to create, list, add, and remove settings securely.
+
+## Usage
+
+```
+opensearch-keystore [command] [options]
+```
+
+### Commands
+
+- `create`: Initializes a new keystore. If a keystore already exists, this command will overwrite it.
+- `list`: Lists all settings in the keystore.
+- `add <setting-name>`: Adds a new setting to the keystore. Prompts for the value to be securely stored.
+- `add-file <file-name>`: Adds a new file to the keystore.
```

Comment:
What type of files would you add to the keystore?

Reply:
Private keys for TLS would be an example I could think of where the contents of the file should be kept secret.

Reply:
Ok, I'm fine with that on lines 68 and 79 as well, then.